src/content/resources/cpp/intro.md

```diff
@@ -5,6 +5,29 @@ created_at: 2019/12/15
 title: C++
 ---
 
-## C++
+## Beginner
 
-The C++ resources ^\_-
+- [Get started](https://isocpp.org/get-started)
+- [Learn C++](http://www.learncpp.com/)
+- [C++ tutorial](http://www.cplusplus.com/doc/tutorial/)
```

Comment:
Maybe keep the domain as the anchor text for these? or something like `Tutorial/learncpp.com` and `Tutorial/cplusplus.com`